indico_toolkit/pipelines/file_processing.py

```diff
@@ -57,7 +57,7 @@ def remove_files_if_processed(self, processed_files: Iterable[str]):
         """
         unprocessed_filepaths = []
         for filepath in self.file_paths:
-            if Path(filepath).name not in processed_files:
+            if self.file_name_from_path(filepath) not in processed_files:
                 unprocessed_filepaths.append(filepath)
         print(f"Removing {len(self.file_paths) - len(unprocessed_filepaths)} files from file_paths")
```

Comment:
at some point, we're going to have a lot of print statements.  should Indico Toolkit be generating it's own logs?

Reply:
yeah I'd like to do that- will make it a separate PR later on